platforms/Windows/sdk-x86.wxs

```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_dispatch" Guid="c3048f41-e208-45ff-a88f-682ff8292eb8">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\group.h" Checksum="yes" />
```

Comment:
Sorry for the random drive-by comment, but I think this ought to be
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\generic_base.h" Checksum="yes" />
```

Reply:
Close, but no dice, mostly because you didn't point out the other mistake in this 😂.  I messed up the location `os/generic_base.h`.  Thanks though!